packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -105,7 +108,16 @@ export const MessageStatusIndicator = (props: MessageStatusIndicatorProps): JSX.
       );
     case 'seen':
       return (
-        <TooltipHost content={strings.seenTooltipText}>
+        <TooltipHost
+          content={
+            messageThreadReadCount === 1 || messageThreadParticipantCount === 1
```

Comment:
If participants are more than 1, message is at seen status, but the read count is only 1, it means we have a bug (read count is defaulted to be 1 since I always read my own message)